org.flightgear.FlightGear.yaml

```diff
@@ -104,17 +88,17 @@ modules:
   buildsystem: simple
   sources:
   - type: archive
-    url: https://sourceforge.net/projects/flightgear/files/release-2018.3/FlightGear-2018.3.2-data.tar.bz2
-    sha256: 277e817a39edb917976441a808b750ee64be38e077897d0d77c55dfc8eba9640
+    url: https://iweb.dl.sourceforge.net/project/flightgear/release-2020.1/FlightGear-2020.1.3-data.tar.bz2
+    sha256: 2196f4ffcdeaae3a8b0a2b6ebf62a456bfd0198abb6c0619f763204d89804a03
     dest: fgdata
   build-commands:
   - mkdir --parents /app/share/flightgear
   - mv fgdata/* /app/share/flightgear/
     
-- shared-modules/udev/udev-175.json
+- shared-modules/gudev/gudev.json
```

Comment:
I don't think flightgear depends on gudev (as it's some GTK/GLib stuff but flightgear is c++), only libudev. It was removed from the shared modules as it's included in the FDO runtime, from which the KDE runtime inherits. So you can remove this line.